lib/features/websocket/presentation/home_screen.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:asset_tracker/features/websocket/application/socket_cubit.dart';
+import 'package:asset_tracker/features/websocket/domain/currency.dart';
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+@RoutePage()
+class HomeScreen extends StatefulWidget {
+  const HomeScreen({super.key});
+
+  @override
+  State<HomeScreen> createState() => _HomeScreenState();
+}
+
+class _HomeScreenState extends State<HomeScreen> {
+  bool isDisconnected = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Socket Tracker'),
+      ),
+      body: BlocBuilder<SocketCubit, SocketState>(
+        builder: (context, state) {
+          if (state is SocketLoading) {
+            return const Center(child: CircularProgressIndicator());
+          } else if (state is SocketError) {
+            return Center(child: Text('Error: ${state.error}'));
+          } else if (state is SocketConnected) {
+            return const Center(child: CircularProgressIndicator());
+          } else if (state is SocketDataReceived) {
+            isDisconnected = false;
+            return _buildDataReceivedUI(context, state.data);
+          } else if (state is SocketDisconnected) {
+            isDisconnected = true;
+            return _buildDataReceivedUI(context, state.data);
+          } else {
+            return const Center(child: Text('Initializing...'));
+          }
+        },
+      ),
+    );
+  }
+
+  Widget _buildDataReceivedUI(BuildContext context, response) {
+    return Center(
+      child: SingleChildScrollView(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
```

Comment:
localization + SizedBox için EmptySpace adından sınıf oluşturulabilir veya 20 gibi rakamlar vermek yerine Sizes kullanabilirsin.